File: services/passport/src/main/java/nc/unc/cs/services/passport/integration/tax_service/Tax.java
package nc.unc.cs.services.passport.integration.tax_service;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;

import javax.persistence.Temporal;
import javax.persistence.TemporalType;
import java.util.Date;

@Data
public class Tax {
    private Long taxId;

    private Integer taxAmount;

    private Boolean status;

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd HH:mm:ss")
    @Temporal(value = TemporalType.TIMESTAMP)
    private Date creationDate;

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd HH:mm:ss")
    @Temporal(value = TemporalType.TIMESTAMP)
    private Date taxPaymentDate;

    private Long citizenId;

    private Long serviceId;
}

File: client-ui/src/models/tax/tax.js
export default class Tax {
    constructor(taxId, taxAmount, status, creationDate, taxPaymentDate, citizenId, serviceId) {
        this.taxId = taxId
        this.taxAmount = taxAmount
        this.status = status
        this.creationDate = creationDate
        this.taxPaymentDate = taxPaymentDate
        this.citizenId = citizenId
        this.serviceId = serviceId
    }
}